CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -0,0 +1,74 @@
+# Release notes for 1.8.0
```

Comment:
I'm confused why this is showing up. Is this a github bug?

https://github.com/kubernetes-csi/external-resizer/blob/release-1.8/CHANGELOG/CHANGELOG-1.8.md